OSSM-9084 Create assembly for supported versions table (#90911)
openshift/openshift-docs: modules/ossm-release-notes-supported-versions.adoc

[id="service-mesh-product-supported-versions_{context}"]
= {SMProduct} supported versions

> This table provides guidance on supported versions in {SMProduct} 3.

Reviewer: ```suggestion
See this following table to know supported versions in {SMProduct} 3.
```

- Suggested an optional change for a more direct language
